index.js

```diff
@@ -55,7 +55,7 @@ var getKeyLink = function(key) {
 };
 
 var printSummary = function(hash, title, method) {
-  var res = '\n\n' + chalk[method](title + ':\n');
+  var res = '\n\n' + chalk[method](title + ':') + chalk.white('\n');
```

Comment:
Using plain '\n' doesn't preserve new lines when it is printed with gulp, I assume gulp is trimming the log before outputting it.